### PR TITLE
Docs tweaks

### PR DESCRIPTION
**Pull request by Hotell**
## Requirements

* [ ] Read the
      [contribution guidelines](https://github.com/skatejs/skatejs/blob/5.x/CONTRIBUTING.md).
* [ ] Wrote tests.
* [x] Updated docs and upgrade instructions, if necessary.
* [ ] Updated TS definitions, if necessary.

## Rationale

- Update docs to latest API's
- ignore site and tests from flow
- closes https://github.com/skatejs/skatejs/issues/1249

### CODE REVIEW

**Comment by Hotell** [comment]
there are still Flow errors within `/src`  this should be addressed in separate PR
image

**Comment by Hotell** [comment]
Flow errors fixed

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 _Delete this section if this is a feature request._
 
 * Skate version: _x.x.x_
+  <!--(run `npm list skatejs` from a terminal/cmd prompt and paste output below): -->
```
Comment (treshugart) [inline .github/ISSUE_TEMPLATE.md]: This isn't actually displayed. Should probably use italics?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -3,7 +3,10 @@
...
 * Affected browsers (and versions): _IE 10_
+* Flow/Typescript version ( if issue is related to type system ): _Flow 0.59 /
```
Comment (treshugart) [inline .github/ISSUE_TEMPLATE.md]: Super nit: consistent spacing on parenthesis (no spaces).

File: README.md [hunk 3]
```diff
@@ -72,14 +72,8 @@ import { withComponent } from 'skatejs';
...
-  renderer (renderRoot, render) {
-    renderRoot.innerHtml = '';
-    renderRoot.appendChild(render());
-  }
-  render () {
-    const el = document.createElement('span');
-    el.innerHTML = 'Hello, <slot></slot>!';
-    return el;
+  render() {
+    return 'Hello, <slot></slot>!';
```
Comment (treshugart) [inline README.md]: I meant to discuss this with you. Do you think this is a valuable change? It felt very right to do.

**Comment by treshugart** [comment]
I left some comments. Only real remaining issue is to move the changes from `src/emit.js` to `packages/skatejs/src/emit.js`. If you want, I can do that. I'll assume you're doing it unless you mention me about it. Thanks!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Hotell) [reply .github/ISSUE_TEMPLATE.md]: nope we as maintainers don't wanna get disturbed by reading that :) it's just helper info for the issuer

File: README.md (hunk 3 above)
Comment (Hotell) [reply README.md]: yup it's definitely good to have some default just by installing skate with any deps :) 👍

**Comment by Hotell** [comment]
squashed /rebased / conflicts resolved - re-review -> merge if no further comments. thx @treshugart

**Pull request merged**